src/controller.rs

```diff
@@ -331,7 +331,8 @@ fn dispatch_server_request(request: MethodCall, ctx: &mut Context) {
             workspace::apply_edit_from_server(request.id, request.params, ctx);
         }
         request::WorkspaceConfiguration::METHOD => {
-            workspace::configuration(request.id, request.params, ctx);
+            let result = workspace::configuration(request.params, ctx);
+            ctx.reply(request.id, result);
```

Comment:
Now we send a `ctx.reply` in every branch but the one in `apply_edit_from_server` is a bit hidden, we should make that consistent. Maybe even by sharing a single `ctx.reply` for all branches.
(Not necessary to do that now, I can do the cleanup)